CMSgov/cms-carts-seds
# Add seed data for 2022 AL

### Description
See title


### Related ticket(s)


CMDCT-2216

---
### How to test

Run CARTS
Verify when you login as stateuser2@test.com reports for 2020, 2021, and 2022 show up
If you open 2022 there are many fields already filled in
No breaking errors as you go through the report

---
### Author checklist


---
